src/phylo/distances.jl

```diff
@@ -0,0 +1,172 @@
+# phylo/distances.jl
+# ==================
+#
+# Types and methods for computing evolutionary and genetic distances.
+#
+# Part of the Bio.Phylo module.
+#
+# This file is a part of BioJulia. License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+
+# Types
+# -----
+
+# Evolutionary distances
+abstract EvoDist
```

Comment:
If you name it `EvolutionaryDistance` you can remove the comment above 😄 . I think it is no use in abbreviating "Distance" as "Dist"; it saves four types but that's all.